README.rst

```diff
@@ -490,6 +488,13 @@ Similarly, in production, the "mfe" Docker image will be rebuilt automatically d
     EOF
     docker buildx create --use --name=singlecpu --config=./buildkitd.toml
 
+Deploying Changes to Production
+-------------------------------
+
+You will need to rebuild the mfe Docker image with your changes, and then deploy it to production - exactly how depends on whether you are running ``tutor local`` or ``tutor k8s``.
```

Comment:
I think this section can be removed. The next section has the appropriate details (the codebase is on same machine, build image, and the restart). We can mention the production environment (k8s vs local) in that section.

Reply:
@DawoudSheraz Per discussion in https://discuss.openedx.org/t/operation-of-the-mfe/15396/5 people are getting stuck here. I don't think removing this is the right call.

Reply:
I believe this adds relevant information with more visibility, in a section with an anchor so we can refer people directly to it.  I agree with @sarina that this questions comes up often enough that it warrants making it more easily accessible.

Reply:
Oh, I did not mean removing the entire section. I meant the first paragraph (🤦🏽 ). The second paragraph includes the necessary details:
- The codebase is on the same machine
- Building the image
- Start after build

The only missing item is the selection of local vs k8s, which is mentioned in the first paragraph.  That can be added to 2nd one.

Reply:
You're suggesting to remove this sentence (which provides context to people who already know what they're doing) and only have the second sentence? I disagree - I think the context is helpful here, and more explicit. Docs should be easy to read and follow, they do not need to be the most concise thing.